### ISSUE TITLE
Bug: Symlinks not created (for namespaced packages?)

### ISSUE THREAD

**Issue by joscha** [yarnpkg/yarn#950]
**Do you want to request a _feature_ or report a _bug_?**

bug

**What is the current behavior?**

``` console
yarn add @atlassian/prebake-distributor-runner@1.0.2
yarn add v0.15.1
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
warning benchmark@1.0.0: The engine "rhino" appears to be invalid.
warning @atlassian/prebake-distributor-runner@1.0.2: The engine "mvn" appears to be invalid.
warning @atlassian/prebake-distributor-runner@1.0.2: The engine "java" appears to be invalid.
[3/4] 🔗  Linking dependencies...
warning Unmet peer dependency "babel-template@^6.2.0".
warning Unmet peer dependency "babel-types@^6.2.0".
error ENOENT: no such file or directory, symlink '../../@atlassian/prebake-distributor-runner/index.js' -> '/Users/jfeth/dev/atlaskit/node_modules/.bin/@atlassian/prebake-distributor-runner'
    at Error (native)
info Visit http://yarnpkg.com/en/docs/cli/add for documentation about this command.
```

**If the current behavior is a bug, please provide the steps to reproduce.**

Run `yarn add @atlassian/prebake-distributor-runner@1.0.2`

**What is the expected behavior?**

That the symlink gets generated correctly 

**Please mention your node.js, yarn and operating system version.**
node 6.5.0
yarn 0.15.1
OSX 10.12 (but also happens in an alpine 3.4)

**Comment by joscha** [comment]
I suspect the namespacing to be part of the problem (unconfirmed).

**Comment by kasperlewau** [comment]
This looks like a duplicate of https://github.com/yarnpkg/yarn/issues/759 

Fwiw, I'm trying to `yarn install` with the addition of `mkdirp` during the link process (PR: https://github.com/yarnpkg/yarn/pull/910) but so far no luck on getting my scoped bin files linked correctly.

**Comment by singuerinc** [comment]
@joscha @kasperlewau I had the same issue (#1161) but now it works.

**Comment by busches** [comment]
Can you reproduce this with `yarn@latest`? I'm not able to recreate the issue using the latest Yarn `v0.17.10`.

**Comment by singuerinc** [comment]
I think this issue should be closed since is working fine from v0.16.1
